src/seismometer/configuration/export_config.py

```
@@ -32,6 +35,11 @@ def __init__(self, raw_config: dict):
         else:
             self.otel_stdout = False
 
+        if "hostname" in log_config:
+            self.hostname = log_config["hostname"]
+        else:
+            self.hostname = "otel-collector"
```

rather than default, can you instead throw a ValueError?